app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala

```diff
@@ -52,7 +65,11 @@ class TaxReturnsConnector @Inject() (
     val timer = metrics.defaultRegistry.timer("ppt.returns.create.timer").time()
     httpClient.POST[TaxReturn, TaxReturn](appConfig.pptReturnsUrl, payload)
       .andThen { case _ => timer.stop() }
-      .map(response => Right(response.toTaxReturn))
+      .map {
+        response =>
+          logger.info(s"Create ppt tax returns had response [${toJson(response)}]")
```

Comment:
Again - is the response the whole return?  Without knowing what is in a return it's hard to comment on whether we should be logging it